server/views/rivers-and-sea.html

```diff
@@ -17,65 +16,82 @@
 {% endblock %}
 
 {% block content %}
-  <div class="govuk-width-container">
-    <div class="govuk-grid-row">
-      <div class="govuk-grid-column-full">
-        <h1 class="govuk-heading-l govuk-!-padding-top-0 h1-gap">
-          Rivers and the sea: understand your flood risk
-        </h1>
-      </div>
+
+<nav aria-label="Pages with more detail on this service" class="gem-c-contents-list govuk-!-margin-bottom-3"
+role="navigation">
+<ol class="gem-c-contents-list__list">
+  {% if surfaceWaterIsFirst %}
+  {% include 'partials/nav-sw.njk' %}
+  {% endif %}
+  {% include 'partials/nav-rs.njk' %}
+  {% if not surfaceWaterIsFirst %}
+  {% include 'partials/nav-sw.njk' %}
+  {% endif %}
+  {% include 'partials/nav-gw.njk' %}
+  </ol>
+</nav>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible govuk-!-margin-top-6">
+      <h1 class="govuk-heading-xl govuk-!-margin-top-6">
+        Rivers and the sea: understand your flood risk
+      </h1>
+      <p class="govuk-body"><strong>Your selected location:</strong> {{ fullAddress }}</p>
+      <p class="govuk-body">This information tells you the flood risk of the land around a building, not the building 
+        itself.</p>
     </div>
+  </div>
   
-    <nav aria-label="Pages with more detail on this service" class="gem-c-contents-list govuk-!-margin-bottom-3"
-      role="navigation">
-      <ol class="gem-c-contents-list__list">
-        {% if surfaceWaterIsFirst %}
-        {% include 'partials/nav-sw.njk' %}
-        {% endif %}
-        {% include 'partials/nav-rs.njk' %}
-        {% if not surfaceWaterIsFirst %}
-        {% include 'partials/nav-sw.njk' %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h2 class="govuk-heading-l govuk-!-padding-top-6 govuk-!-margin-bottom-4">How likely a river or sea flood is</h2>
+      <p class="govuk-body govuk-!-padding-top-4">The yearly chance of rivers and sea flooding is:</p>
+      <p class="govuk-body">
+        <strong class="govuk-tag govuk-tag--{{ riversSeaRiskStyle }} label-mod-depth">
+          <span>{{ riverAndSeaRisk }}</span>
+        </strong>
+        {% if riverAndSeaRisk == riverAndSeaRiskCC %}
```

Comment:
There seems to be an issue with this section, when I use the postcode WA4 1AB I get the following:
![Screenshot 2024-10-16 at 12 06 40](https://github.com/user-attachments/assets/1d1f7c73-75e5-4c6c-90d1-3c577dda640a)

I dont think it should be reducing and if it was it should also say 'decreasing' or 'reducing' depending on what the ACs say.